stdlib/tkinter/__init__.pyi

```diff
@@ -3264,7 +3296,7 @@ class PanedWindow(Widget):
     @overload
     def configure(self, cnf: str) -> Tuple[str, str, str, Any, Any]: ...
     config = configure
-    def add(self, child, **kw): ...
+    def add(self, child: Widget, **kw): ...
```

Comment:
For future reference, `Widget` doesn't actually mean any tkinter widget, like you would expect, but rather any widget that can be added into another widget. So in this case `Widget` is the correct type, but often it isn't.